src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -94,7 +94,8 @@ export const MyVideosView = () => {
       return
     }
     addVideoTab({ id, isDraft: opts.draft })
-
+    const tabIdx = videoTabs.findIndex((t) => t.id === id)
+    if (tabIdx >= 0) setSelectedVideoTabIdx(tabIdx)
```

Comment:
We probably should call `addVideoTab` only if the tab is not found right?

Reply:
Not necessarily as the `addVideoTab` fn already checks for that so would be redundant